v1/s012-pyglet/pong.html

```diff
@@ -0,0 +1,616 @@
+                        správně a na správném místě? Princip určení souřadnic
+                        je podobný jako u vykreslení míčku.
+<details class="solution notes"><h4>Řešení</h4>
+<pre>
+def vykresli():
+    ...
+    # palky - udelame si seznam souradnic palek, a pro kazdou dvojici souradnic
+    # v tom seznamu palku vykreslime
+    for x, y in [(0, pozice_palek[0]), (SIRKA, pozice_palek[1])]:
+        nakresli_obdelnik(
+            x - TLOUSTKA_PALKY,
+            y - DELKA_PALKY // 2,
+            x + TLOUSTKA_PALKY,
+            y + DELKA_PALKY // 2,
+        )
+</pre>
+</details>
+                    </div>
+                    <div>
+                        Přehlednosti hry pomůže <strong>půlící čára</strong>
+                        uprostřed. Jak ji ale namalovat?
+                        Nebudeme vymýšlet zbytečné složitosti.
+                        Namalujme ji jako sérii obdélníčků táhnoucích se odshora
+                        dolů. Chce to jen vygenerovat seznam souřadnic,
+                        které budou mít dostatečné rozestupy, a na každé
+                        z nich vykreslit obdélníček. Kterou funkci z Pythonu
+                        bys použila na získání tohoto seznamu souřadnic?
+<details class="solution notes"><h4>Řešení</h4>
+<pre>
+def vykresli():
+    ...
+    # prerusovana pulici cara - slozena ze spousty malych obdelnicku
+    for y in range(DELKA_PULICI_CARKY // 2, VYSKA, DELKA_PULICI_CARKY * 2):
+        nakresli_obdelnik(
+            SIRKA // 2 - 1,
+            y,
+            SIRKA // 2 + 1,
+            y + DELKA_PULICI_CARKY
+        )
+</pre>
+</details>
+                    </div>
+                    <div>
+                        Co nám ještě chybí? <strong>Počítadlo skóre</strong>
+                        pro oba hráče.
+                        K tomu se musíme naučit vykreslovat v Pygletu text.
+                        V Pygletu je modul <code>text</code>, který obsahuje
+                        objekt Label (Nápis). Ten se hodí k vykreslení hodnoty
+                        skóre. Objekt musíme nejdřív vytvořit. To uděláme 
+                        kulatými závorkami za jménem objektu, jako bychom
+                        volali funkci a uložíme si ho do proměnné:
+                        <code>napis = Label()</code>. Normálně bychom objekt
+                        vytvořili jen jednou a pak měnili jeho hodnotu, ale
+                        pro jednoduchost vytvoříme vždy nový a celé to zabalíme
+                        do funkce. V jejím závěru musíme na nadpisu zavolat
+                        metodu <code>draw()</code>, jinak se nápis nevykreslí.
+
+<pre>
+def nakresli_text(text, x, y, pozice_x):
+    """Nakresli dany text na danou pozici
+
+    Argument ``pozice_x`` muze byt "left" nebo "right", udava na kterou stranu
+    bude text zarovnany
+    """
+    napis = pyglet.text.Label(
+        text,
+        font_name='League Gothic',
```

Comment:
Chtělo by to do instrukcí přidat, aby si všichni tenhle font stáhli. A udělat s ním i screenshot :)
(Případně sehnat lepší (hranatější) open-source font? Na Google Fonts možná bude.)